jobs/src/test/scala/dcos/metronome/jobspec/impl/JobSpecSchedulerActorTest.scala

```diff
@@ -67,6 +67,21 @@ class JobSpecSchedulerActorTest extends TestKit(ActorSystem("test")) with FunSui
     system.stop(actor)
   }
 
+  test("If the next scheduled time has reached AND FORBID, a new job run is NOT triggered however a new time is scheduled") {
```

Comment:
is this test still passing? it"s weird because we moved the logic to other actor...